docs/_posts/2018-01-15-jekyll-3-7-1-released.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Jekyll 3.7.1 Released"
+date: 2018-01-15 11:21:00 +0530
+author: ashmaroli
+version: 3.7.1
+categories: [release]
+---
+
+Close on the heels of shipping 3.7.0, we were informed of a couple of
+regressions due to the changes made in that release. In due time, Team Jekyll
+set out to address those issues as early as possible.
```

Comment:
Link to Jekyll teams ?

Reply:
As Jekyll need more contributors, linking to teams is meant as a way of reminding that people are welcome to join.

Reply:
isn't `Team Jekyll` meant kind of tongue-in-cheek here? at least that's how i interpreted it, but a link to jekyll teams would be nice, agreed

Reply:
I did not intend it to be `tongue-in-cheek` at all..:
I just meant to refer to the maintainers of the codebase without focusing on a particular team or set of individuals..